io/net/virt-net/lpm.py

```diff
@@ -56,10 +56,10 @@ def setUp(self):
         self.hmc_pwd = self.params.get("hmc_pwd", '*', default='********')
         self.options = self.params.get("options", default='')
         self.net_device_type = self.params.get("net_device_type", default='')
-        self.lpar = self.get_partition_name("Partition Name")
+        self.lpar = self.params.get("lpar", default='')
```

Comment:
Thinking about this more, I don't think the FVT team (the team that owns these tests) would want their configuration changed. In other words, I think they would still want their test to be self validated. So I think the logic should be, if the "lpar" parameter is empty then use the get_partition_name function to get the lpar name.

Reply:
Oh and add a self.log.info line stating that the lpar name was not specified, so we are migrating the current LPAR.